### ISSUE TITLE
Notifications have started arriving in Chinese

### ISSUE THREAD

**Issue by apbarratt** [drakhart/ha-super-soco-custom#16]
I don't think this is something that can be fixed on this end, but it seems I'm now receiving all my warning's in Chinese.
Google translate seems to show they are the same "burglar alarm" notifications that I normally get, but obviously they're not being filtered out by my automations anymore as the wording is different.

I only mention it here in case there is some kind of language selection function in the configuration that might have gone wonky.

Let me know if any logs are of interest.

![Screenshot 2024-04-08 at 12 58 16](https://github.com/drakhart/ha-super-soco-custom/assets/642320/b885a90f-e7d9-4ad8-a5b7-36c8426c2f59)
Screenshot shows that the correct language was arriving at 02:33 last night but switched to Chinese about an hour later at 03:34.

I do wish there were a way to figure out which alerts are just vibration alerts and which are strong enough to actually set off the alarm as I note the alarm on the bike does not go off for all of these.  But that's a whole other story and not the point of this ticket :)

**Comment by apbarratt** [comment]
I've checked with a group on Facebook to ask if folk are receiving these on the official app too.  If they are not, it may in fact be that the server will only send out Chinese notifications from now on and is using a locale pack locally on the app to translate them.  If that is the case, we'll need to create a similar locale pack here too, which isn't a terrible idea anyway to be honest as it will allow for expansion into other non-officially supported languages also (and frankly better translations in general) :)

**Comment by apbarratt** [comment]
Seems I'm back to English after about 28 hours.  Most odd.  I still like the idea of a locale pack though just to get better translations, it's a shame we're not getting a universal error format for easy translation though as there must be quite a few languages that server could be spitting out messages in.
![Screenshot 2024-04-09 at 12 32 29](https://github.com/drakhart/ha-super-soco-custom/assets/642320/7179de68-62fd-4549-8b05-288cb134e490)

**Comment by drakhart** [comment]
Sorry for the delay in replying. I did some research on this too, just forgot to post the results here:
- I started to receive notifications in Chinese by that time too, both in this integration and the Vmoto Soco app (the one with dark background).
- I checked with some folks in the Spanish Super Soco Telegram group and some of them confirmed they were receiving notifications in Spanish in the new Vmoto app (the one with white background and aqua blue accent color). I wasn't able to log in with that app, so I couldn't check myself.
- After a few days notifications were back to usual English.

I'll try to decompile Vmoto's new app whenever I have the time and check if there's some mapping between those messages and the user's language, and will report back. If that's the case it would be nice to implement locales for this integration, as you mention. 👍
